mbzirc_ign/src/GameLogicPlugin.cc

```diff
@@ -790,6 +896,208 @@ void GameLogicPluginPrivate::Finish(const ignition::msgs::Time &_simTime)
   this->finished = true;
 }
 
+/////////////////////////////////////////////////
+bool GameLogicPluginPrivate::OnReportTargets(
+    const ignition::msgs::StringMsg_V &_req,
+    ignition::msgs::Boolean &_res)
+{
+  this->reports.push_back(_req);
```

Comment:
Should we use `reportMutex` here? I don't think it's used when reading its value either.

Reply:
oops yeah that was meant to be done here. Added. [de1da17](https://github.com/osrf/mbzirc/pull/52/commits/de1da1726a12615716efd5b1d684e417c7ce8ea8)